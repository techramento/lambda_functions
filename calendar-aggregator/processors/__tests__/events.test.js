const eventsProcessor = require('../events')('MyMeetup')

const description = `The Name of the Meetup
Thursday, January 01 at 7:00 PM

EVENT DESCRIPTION...

https://example.org/`

const MockCalendar = {
  items: [
    {
      description
    }
  ]
}

describe('events processor', () => {
  it('transforms the data of each event item', () => {
    expect(eventsProcessor(MockCalendar)).toMatchSnapshot()
  })
})
