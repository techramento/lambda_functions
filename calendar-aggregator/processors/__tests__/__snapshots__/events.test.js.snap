// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`events processor transforms the data of each event item 1`] = `
Array [
  Object {
    "description": "The Name of the Meetup
Thursday, January 01 at 7:00 PM

EVENT DESCRIPTION...

https://example.org/",
    "meetup": Object {
      "description": "EVENT DESCRIPTION...",
      "group": "The Name of the Meetup",
      "short_name": "MyMeetup",
      "url": "https://example.org/",
    },
  },
]
`;
